drivers/modem/ublox-sara-r4.c

```diff
@@ -1192,7 +1250,7 @@ static void modem_reset(void)
 	} else {
 		/* register operator automatically */
 		ret = modem_cmd_send(&mctx.iface, &mctx.cmd_handler,
-				     NULL, 0, "AT+COPS=0,0",
+				     NULL, 0, "AT+COPS=0,2",
```

Comment:
I'm not sure how that affects the code. Could you elaborate what does it do and why is it needed?

Reply:
This option selects the format for the AT+COPS? read command to be numerical, as described here: [https://infocenter.nordicsemi.com/topic/.../cops_read.html](https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/nw_service/cops_read.html)
As the AT+COPS? command was not previously used in the driver, this should not affect existing code.

Reply:
I have now removed that change, and set the format directly before reading. This will avoid interference with other changes in the same drivers.